drivers/serial/uart_mcux_lpuart.c

```diff
@@ -1199,6 +1220,21 @@ static void mcux_lpuart_line_ctrl_set_rts(const struct mcux_lpuart_config *confi
 		config->base->MODIR |= (LPUART_MODIR_TXRTSPOL_MASK | LPUART_MODIR_TXRTSE_MASK);
 	}
 }
+#endif /* LPUART_HAS_MODEM */
```

Comment:
make an #else case taht does like,

#define mcux_lpuart_line_ctrl_set_rts(...)

then you can remove all the ifdefs below, similar for other functions which may or may not be defined, it makes the rest of the code more readable

Reply:
very helpful, thank you.  I used this a couple times for the helper functions